src/components/Header/header.js

```diff
@@ -16,30 +24,53 @@ class Header extends Component  {
         };
 
         this.responsiveUtils = new ResponsiveUtils();
+        
     }
     componentDidMount(){
-
+        
         // On Resize and On Orientation Change Handler added to ResponsiveUtils Object
         this.responsiveUtils.__proto__.onResize = () => {
             this.responsiveUtils.viewPortSelector();
             this.setState ({
-                viewport : this.responsiveUtils.viewPort
+                viewport : this.responsiveUtils.viewport
             })
         }
         // Component starts listening to 'resize' and 'onOrientationChange' events
         this.responsiveUtils.init();
 
-        // Get the value of viewport on component mount
+        // Set the value of viewport on component mount
         this.responsiveUtils.onResize();
+
     }
 
     render(){
+
+        this.imageOptimizer = new ImageOptimize({
+            mobile_image : '../../assets/images/walking-dog-bg-320.png', 
+            large_image : '../../assets/images/walking-dog-bg.png', 
+            viewport : this.state.viewport
+        })
+
+        this.bgImage = {
+            backgroundImage: `url(${this.imageOptimizer.optimizeImage()})`
+        }
+        
         return(
-            <div style={bgImage} className="header">
-                
+            <div style={this.bgImage}   className="header">
+               <Container>
+                   <Row>
+                       <Col xs="3" className="pt-3">
```

Comment:
You are using '' and "" anywhere without any proper rule.? Is this a good approach? You can apply linting to it for better code qualities. Check out ESlint and how AirBNB uses it.